src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializer.java

```diff
@@ -294,12 +298,13 @@ private static String getMrnDate(SyaCaseWrapper syaCaseWrapper) {
                 null;
     }
 
-    private static Address handleAddress(SyaContactDetails contactDetails, boolean isIbc, String party) {
+    private static Address handleAddress(SyaContactDetails contactDetails, String party) {
         Address.AddressBuilder addressBuilder = Address.builder()
             .line1(contactDetails.getAddressLine1())
             .line2(contactDetails.getAddressLine2())
-            .town(contactDetails.getTownCity());
-        if (isIbc && contactDetails.getInMainlandUk() != null) {
+            .town(contactDetails.getTownCity())
```

Comment:
do we need to include addressLine3 or is it not relevant in this context
see bug ticket https://tools.hmcts.net/jira/browse/SSCSCI-1474